### PR TITLE
doc: Add HW getting started guide and minor fixes

### LINKED ISSUE #8: Getting started with the Sensor Edition HW
#### Summary

Write-up about how to get started with the SE board.

#### Why do we need this ?

To be able to easily use the SE board. 

#### What is already there? What do you see now?

Nothing

#### What is missing? What do you want to see?

- Getting started with the tools
- Getting started with the board

#### How do you propose to document this?

With picture and explanation guides

#### Can you do this yourself and submit a Pull Request?

Yes

### PR DESCRIPTION
**Pull request by elsalahy**
#### Summary

Adds a HW getting started.
Closes #8

#### Screenshots

...

#### Changes

- Add HW tools setup and HW pinout explanation
- Fix minor typos and broken links

#### Notes for Reviewers

...

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [x] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by azerimaker — DISMISSED** [review]
Looks good, @elsalahy . Later on will improve the annotated graphics.

File: doc/content/getting-started/se-hw/_index.md [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+
+It is always a good idea to get familiar with the available HW design files. Check out the <a href="https://github.com/TheThingsIndustries/generic-node-se" target="_blank">{{% gnse %}} on Github</a> to get familiar with the repository structure.
+
+Next, clone the repository to get its local copy on your machine by running the following command:
+
+```bash
+$ git clone --branch develop \
+--recurse-submodules \
+https://github.com/TheThingsIndustries/generic-node-se.git
+```
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/_index.md:22]: add a blank line after bash code block

File: doc/content/getting-started/se-hw/_index.md [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+If you are not too familiar with Git, visit the official GitHub documentation to <a href="https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository" target="_blank">learn to clone a repository</a>.
+
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/_index.md:24]: remove one blank line

File: doc/content/getting-started/se-hw/_index.md [hunk 3]
```diff
@@ -1,8 +1,30 @@
...
+
+## Becoming familier with the Generic Node Sensor Edition Board
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/_index.md:26]:
```suggestion
## Becoming familiar with the Generic Node Sensor Edition Board
```

File: doc/content/getting-started/se-hw/_index.md [hunk 4]
```diff
@@ -1,8 +1,30 @@
...
+
+It is a good idea to become familiar with the {{% gnse %}} on-board components and peripherals, see the [Sensor Edition Board]({{< ref "/hardware/se-board" >}}) guide.
+
+Next, we setup the hardware design tools to be able to explore further the schematics and PCB layouts, see [Sensor Edition Hardware Design Tools]({{< ref "/getting-started/se-hw/se-hw-design" >}})
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/_index.md:30]:
```suggestion
Next, follow the [Sensor Edition Hardware Design Tools]({{< ref "/getting-started/se-hw/se-hw-design" >}}) section to setup the hardware design tools in order to further explore the schematics and PCB layouts.
```

File: doc/content/getting-started/se-hw/se-hw-design/_index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Sensor Edition Hardware Design Tools"
+description: ""
+weight: -1
+distributions: null
+---
+Setup the tools and explore the reference hardware design.
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/se-hw-design/_index.md:7]:
```suggestion
This section helps you to setup the hardware design tools and explore the reference hardware design.
```

File: doc/content/getting-started/se-hw/se-hw-design/_index.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+<!--more-->
+
+## Prerequisites
+
+To be able to view the schematics and PCB layout, you need to install the necessary tools first:
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/se-hw-design/_index.md:13]: remove this, no need to explain, it's prerequisites

File: doc/content/getting-started/se-hw/se-hw-design/_index.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+1. [KiCAD v.5.1.8](https://kicad.org/)
+2. [FreeCAD 0.18](https://www.freecadweb.org/)
+
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/se-hw-design/_index.md:17]: remove one blank line

File: doc/content/getting-started/se-hw/se-hw-design/_index.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+## Setup Example
+
+1. Open KiCad and press **Open existing project**
+2. Navigate to `generic-node-se/Hardware/pcb-stm32wl/` and choose `pcb-stm32wl.pro`
+{{< figure src="se-hw-design-kicad.png" alt="KiCad open project" >}}
+
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/se-hw-design/_index.md:24]: remove one blank line

File: doc/content/getting-started/se-hw/se-hw-design/_index.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+1. [KiCAD v.5.1.8](https://kicad.org/)
+2. [FreeCAD 0.18](https://www.freecadweb.org/)
+
+
+## Setup Example
+
+1. Open KiCad and press **Open existing project**
+2. Navigate to `generic-node-se/Hardware/pcb-stm32wl/` and choose `pcb-stm32wl.pro`
+{{< figure src="se-hw-design-kicad.png" alt="KiCad open project" >}}
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/se-hw-design/_index.md:23]:
```suggestion
1. Open KiCad and press the **Open existing project** button.
2. Navigate to `generic-node-se/Hardware/pcb-stm32wl/` and choose `pcb-stm32wl.pro`.
{{< figure src="se-hw-design-kicad.png" alt="KiCad open project" >}}
```

File: doc/content/getting-started/se-hw/se-hw-design/_index.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Exploring the Schematics
+
+1. Double click on the `pcb-stm32wl.sch`
+{{< figure src="se-hw-design-sch.png" alt="KiCad open schematics" >}}
+
+2. You should be able to view and explore the schematics design
+{{< figure src="se-hw-design-sch-preview.png" alt="KiCad schematics view" >}}
+
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/se-hw-design/_index.md:33]: remove blank line

File: doc/content/getting-started/se-hw/se-hw-design/_index.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+{{< figure src="se-hw-design-kicad.png" alt="KiCad open project" >}}
+
+
+## Exploring the Schematics
+
+1. Double click on the `pcb-stm32wl.sch`
+{{< figure src="se-hw-design-sch.png" alt="KiCad open schematics" >}}
+
+2. You should be able to view and explore the schematics design
+{{< figure src="se-hw-design-sch-preview.png" alt="KiCad schematics view" >}}
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/se-hw-design/_index.md:32]:
```suggestion
1. Double click on the `pcb-stm32wl.sch`.
{{< figure src="se-hw-design-sch.png" alt="KiCad open schematics" >}}

2. You should now be able to view and explore the schematics design.
{{< figure src="se-hw-design-sch-preview.png" alt="KiCad schematics view" >}}
```

File: doc/content/getting-started/se-hw/se-hw-design/_index.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Exploring the PCB layout
+
+1. Double click on the `pcb-stm32wl.kicad_pcb`
+{{< figure src="se-hw-design-pcb.png" alt="KiCad open schematics" >}}
+
+2. You should be able to view and explore the schematics design
+{{< figure src="se-hw-design-pcb-preview.png" alt="KiCad schematics view" >}}
```
Comment (nejraselimovic) [inline doc/content/getting-started/se-hw/se-hw-design/_index.md:41]:
```suggestion
1. Double click on the `pcb-stm32wl.kicad_pcb`.
{{< figure src="se-hw-design-pcb.png" alt="KiCad open schematics" >}}

2. You should be able to view and explore the schematics design.
{{< figure src="se-hw-design-pcb-preview.png" alt="KiCad schematics view" >}}
```

**Review by nejraselimovic — APPROVED** [review]
Minor stuff, you can merge

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**